celobox/passwd.py

```diff
@@ -31,7 +34,9 @@ def new_session(self):
 
     def load_data(self, domain):
         try:
-            return json.load(open('manifests/%s.json' % domain))
+            return json.load(open(
+                resource_filename(__name__, os.path.join('manifests', '{}.json'.format(domain)))
```

Comment:
This is going to need to change/be updated with the branch that allows a YAML specification, no? Did you rebase it incorrectly?